packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -128,13 +146,41 @@ function extractTime(strToParse) {
 	return [timeValidated];
 }
 
-const PROPS_TO_OMIT_FOR_INPUT = ['selectedDateTime', 'onChange', 'onError'];
+function computeDateRelatedState(selectedDateTime) {
+	const isDateTimeValid = isDateValid(selectedDateTime);
+
+	if (selectedDateTime !== undefined && isDateTimeValid) {
```

Comment:
it's not covered because I considered an `undefined` Date to be valid (ie. a date not defined is valid).
So here I checked it's a `defined valid date`

Reply:
✔